data/energy/oil-gas/index.html

```diff
@@ -17,11 +17,11 @@
     - Nightly
 OilandGasFields:
   hub:
-    name: Utah Oil Gas Fields
+    name: Utah Oil and Gas Fields
```

Comment:
**praise**: nice catch on the wrong name

Reply:
thanks, that one took me a min.